earth_enterprise/tools/shunit2-2.1.6/bin/gen_test_results.sh

```diff
@@ -0,0 +1,62 @@
+#! /bin/sh
```

Comment:
Is this not available as an OS level package we could add as a prereq? I really hate bundling new stuff

Reply:
Well, there is https://github.com/bpkg/bpkg#usage. I couldn't tell if it supports updating already installed packages, and we might have to contribute a packaging for ShUnit 2 to be able to use it with `bpkg`.

Otherwise, we could just add the ShUnit 2 repository as a submodule of the Open GEE repository. It may be less instrumentalized, but that way it would be automatically fetched when someone clones the Open GEE repository, and it would be kept up to date.

Reply:
Looks like it's included in the default yum and apt repos.  I'll see about switching to those.

Reply:
So this is available for RHEL and CentOS 6 but not 7.  I could switch to another framework but I really like the way this one works.  Perhaps this is something we should discuss.